test/index.ts

```diff
@@ -982,11 +982,15 @@ describe('Spanner', () => {
       });
     });
 
-    it('should continue if do not have permission for instance', done => {
+    it('should continue if a permission error is returned for the instance.', done => {
+      const PERMISSION_DENIED_MESSAGE = `The client library attempted to connect to 
```

Comment:
It would be great if there was a way to share this constant across the two index.ts files so that you don't have to repeat this large block of text.